Salamanders

MintyChip opened the pull request:
<b>Please only submit Pull Requests to the dev branch!</b>

### Things to include in a large Pull Request: ###

- What is the purpose of the pull request?
    Add a salamander race with two subspecies

- Give a brief description of what you changed or added.
1. Adds a salamander race with two subspecies: flame (mainly desert/volcano) and frost (snow/mountains)
2. Food item: apple pie à la mode
3. Drink items: iced tea and hot cocoa
4. combat move: exposure therapy
5. two colours

NOTE: I based the frost salamanders are based off the Frostscale Salamanders from CoCII. If you'd rather not step on any toes, I can switch the aurora on the end of the frost salamanders to a cold mist. 

- Are any new graphical assets required?
All assets are included

- Has this change been tested? If so, mention the version number that the test was based on.
0.4.5.5
- So we have a better idea of who you are, what is your Discord Handle?
MintyChip#1944
PS - I removed #1705 because it was branched off of #1658. This one is not dependent on that pr.
- If you want quick feedback, you can use @Innoxia, or jump on the Lilith's Throne discord and send Innoxia a PM.

MintyChip committed:
add Salamanders

Stadler76 commented on res/race/mintychip/salamander/bodyParts/vagina.xml:
```diff
@@ -0,0 +1,72 @@
+	<pubicHairAllowed>true</pubicHairAllowed>
+	
+	<!-- Whether this vagina type lays eggs (true) or gives birth to live young (false). -->
+	<eggLayer>false</eggLayer>
```
```suggestion
	<eggLayer>true</eggLayer>
```

I assume, this was an oversight? Or is it intended that they are mammals?

MintyChip committed:
set egglayer to true

Co-authored-by: Christian Stadler <Stadler76@users.noreply.github.com>

MintyChip committed:
remove extra hair

I removed beards, pubic, armpit, and ass hair

MintyChip committed:
Merge remote-tracking branch 'origin/salamander2' into salamander2

Stadler76 reviewed (approved):
Approving the review request (saw your DM on Discord).
PS: Nice work to remove their hair.

MintyChip committed:
add bookEntries

for some reason, some files weren't transferred correctly, but I think I found all of them now

MintyChip committed:
fix salamander icon

smoothed out a corner in the icon

exposure therapy icon was made easier to read

Innoxia commented:
Thank you for this! 🦎
